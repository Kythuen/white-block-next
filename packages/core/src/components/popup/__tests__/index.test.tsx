import { describe, expect, it } from 'vitest'
import { mount } from '@vue/test-utils'
import Popup from '../popup.vue'
// import { usePopup } from '../use'

describe('# Popup', () => {
  describe('## Base', () => {
    it('### default config', () => {
      const wrapper = mount({
        render() {
          return <Popup />
        }
      })
      // expect(wrapper.classes()).toEqual(['wb-popup'])
      // expect(wrapper.props()).toEqual({})
      expect(wrapper.element).toMatchSnapshot()
    })
  })

  describe.skip('## Slots', () => {})

  describe.skip('## Events', () => {})

  describe.skip('## Props', () => {})

  describe.skip('## Composables', () => {})
})
