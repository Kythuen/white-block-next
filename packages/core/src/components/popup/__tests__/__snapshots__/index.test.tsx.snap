// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`# Popup > ## Base > ### default config 1`] = `
<div
  class="wb-popup-wrapper"
>
  <div
    class="wb-popup-trigger"
    cursor="pointer"
    select="none"
  >
    <!---->
  </div>
  <!--teleport start-->
  <!--v-if-->
  <!--teleport end-->
</div>
`;
